### DISCUSSION TITLE
Running DeePMD enabled LAMMPS with PLUMED?

### DISCUSSION THREAD

**Discussion by kuntalg97** [deepmodeling/deepmd-kit#3197] [Q&A]
Hi,

I've installed DeePMD using Conda, and I've been using the LAMMPS binary that comes along with it. However, I also wish to use PLUMED (along with the deepmd fix in LAMMPS). The LAMMPS binary works well, but I keep getting the following error when I try to use the plumed fix:

+++ Loading the PLUMED kernel runtime +++
+++ PLUMED_KERNEL="/Softwares/PLUMED/plumed-2.9.0/lib/libplumedKernel.so" +++
+++ An error occurred. Message from dlopen(): libgsl.so.23: cannot open shared object file: No such file or directory +++
+++ This error is expected if you are trying to load a kernel <=2.4
+++ Trying /Softwares/PLUMED/plumed-2.9.0/lib/libplumed.so +++
+++ An error occurred. Message from dlopen(): libplumedKernel.so: cannot open shared object file: No such file or directory +++

How do I fix this? Note that lmp -h lists PLUMED as one of the enabled packages, and I've already checked discussions #280 and #2995.

Thanks!

**Comment by njzjz** [comment]
It seems that the lammps loaded the plumed library that is not shipped by conda. You may check your environment variables. If you add the directory of plumed to the environmental variables, you also need to add its dependencies.

Btw, I think LAMMPS hasn't supported plumed 2.9 yet. (It checks whether the plumed version is compatible)

**Reply by kuntalg97** [reply]
I also have a PLUMED 2.4 installed, but I'm not able to invoke it through the LAMMPS included in the conda DeePMD installation. And could you be a bit more specific about how I can include the directory of plumed to the environment variables?

**Reply by njzjz** [reply]
Check `LD_LIBRARY_PATH` and see if it includes the path of plumed but not its dependencies.
